src/app/upload/shared/audio-duration.component.css

```diff
@@ -0,0 +1,11 @@
+button {
+  background: transparent url('/assets/images/info-circle.svg') 50% no-repeat;
+  background-size: cover;
+  height: 18px;
+  width: 18px;
+  margin: 0;
+  vertical-align: text-bottom;
+}
+button:hover, button:focus {
+  background-image: url('/assets/images/info-circle-dark.svg')
+}
```

Comment:
I woulda had you put this in the styleguide but I'm gonna go ahead an look the other way on it for now. ;)